src/samplers/aliastable.jl

```diff
@@ -1,21 +1,20 @@
-struct AliasTable{S} <: Sampleable{Univariate,Discrete}
+struct AliasTable{N} <: Sampleable{Univariate,Discrete}
```

Comment:
I don't think it's worth introducing a type parameter for the size here since it's no longer used in any of the fields. It should be fine with the old `n` version of the definitions below.

Reply:
Made n a field instead of a type parameter